pages/api/courses/index.ts

```diff
@@ -0,0 +1,31 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import prisma from "../../../lib/prisma";
+
+export default async function handle(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === "GET") {
+    const courses = await prisma.course.findMany({});
+    const courseObjects = await Promise.all(
```

Comment:
Love how magical (and clean) `Promise.all` is :)

Reply:
I like it too! Although at some point we'll probably want to wrap things in try catches or handle errors instead of just assuming everything will work